aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -495,6 +495,8 @@ TORCH_META_FUNC(triangular_solve)(const Tensor& self, const Tensor& A, bool uppe
     // no broadcasting for non-strided layout
     set_output_raw_strided(0, self.sizes(), {}, self.options(), {}); // make row major strides for Sparse BLAS
     set_output_raw_strided(1, {0}, {}, self.options(), {}); // return 0-sized tensor
+  } else if (A.layout() == Layout::SparseCsc) {
+      TORCH_CHECK(false, "torch.triangular_solve: unsupported sparse layout.");
```

Comment:
```suggestion
      TORCH_CHECK_VALUE(false, "torch.triangular_solve: unsupported sparse layout.");
```
should be a ValueError, no?

Reply:
Yes, you are right. I will commit the suggested change.